cc2.1/build_cc/ubuntu/Dockerfile

```diff
@@ -0,0 +1,71 @@
+#  This file is part of cc-oci-runtime.
```

Comment:
Could this path please be `2.1/build/ubuntu/Dockerfile` ? the `cc` prefix and suffix is redundant, we know we're building Clear Containers :)